packages/schema/src/config/vite.ts

```diff
@@ -70,6 +70,14 @@ export default defineUntypedSchema({
       emptyOutDir: false
     },
     server: {
+      watch: {
+        ignored: {
+          $resolve: async (val, get) => [
+            await get('analyzeDir'),
+            ...val ?? []
+          ]
+        }
+      },
```

Comment:
I don't think we need to add this here, as it should be present from:

https://github.com/nuxt/nuxt/blob/9878c5b6e4b3d7241910a347412a0936dbc6a3cf/packages/vite/src/vite.ts#L88-L96

Reply:
```suggestion
```

Reply:
Yeah, good point, I forgot to remove it, Thanks!